src/pages/ui-update-sample.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-03-08
+modDate: 2024-05-10
+title: Tenant variables
+description: Tenant variables allow you to specify variables that can have different values for each tenant.
+navSearch: false
+navSitemap: false
+navMenu: false
+robots: noindex, follow
+---
+import TenantsWhichVariableTemplatesApply from 'src/shared-content/tenants/tenants-which-variable-templates-apply.include.md';
+
+Tenant variables allow you to configure [variables](/docs/projects/variables/) that can have different values for each [tenant](/docs/tenants)/environment combination. 
+
+:::figure
+![](/docs/getting-started/tenant-variables.png "width=500")
```

Comment:
As this is a sample, could we add alt text between the `[]` to encourage good behavior?